src/transforms/lua/v2/interop/util.rs

```diff
@@ -77,3 +77,19 @@ where
     }
     Ok(seq)
 }
+
+// Taken from https://github.com/amethyst/rlua/blob/v0.17.0/src/value.rs#L52-L61
+pub fn type_name(value: &LuaValue) -> &'static str {
+    match *value {
+        LuaValue::Nil => "nil",
+        LuaValue::Boolean(_) => "boolean",
+        LuaValue::LightUserData(_) => "light userdata",
+        LuaValue::Integer(_) => "integer",
+        LuaValue::Number(_) => "number",
+        LuaValue::String(_) => "string",
+        LuaValue::Table(_) => "table",
+        LuaValue::Function(_) => "function",
+        LuaValue::Thread(_) => "thread",
+        LuaValue::UserData(_) | LuaValue::Error(_) => "userdata",
+    }
+}
```

Comment:
It's a bit weird that they publicly expose the [`Error`](https://github.com/amethyst/rlua/blob/v0.17.0/src/error.rs#L63-L71) type with a `from` field taking `&'static str`, but not the [`type_name`](https://github.com/amethyst/rlua/blob/v0.17.0/src/value.rs#L50-L63) implementation.

Reply:
Hmm, that is odd. Maybe worth an issue on the crate?